src/index.html

```diff
@@ -12,7 +12,54 @@
       href="./style.css"
     />
```

Comment:
According to the checklist, if the HTML-element has long attribute values or number of attributes is more than 2, you should start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one. Please adjust the formatting accordingly.